instrumentation/lettuce/lettuce-5.1/library/src/main/java/io/opentelemetry/instrumentation/lettuce/v5_1/OpenTelemetryTracing.java

```diff
@@ -178,18 +189,26 @@ private static class OpenTelemetrySpan extends Tracer.Span {
     private final Context context;
     private final SpanBuilder spanBuilder;
     private final RedisCommandSanitizer sanitizer;
+    private final OperationListener metrics;
 
     @Nullable private String name;
     @Nullable private List<Object> events;
     @Nullable private Throwable error;
     @Nullable private Span span;
+    private long spanStartTime;
```

Comment:
(this is the convention we generally use for nano time)
```suggestion
    private long spanStartNanos;
```